### PR TITLE
Start story-cleanup script

### PR DESCRIPTION
**Pull request by daved**
https://www.pivotaltracker.com/story/show/174891360

### CODE REVIEW

**Comment by daved** [comment]
Notes:
1. The amount of stories handled per run is limited by the API or a hardcoded value of 50 (whichever is less). I picked this value because it seems like it would cover most weekly/monthly needs, but minimize affected stories if some mistake occurs.
2. The secret which defines the primary project id will need to be updated for the script to work: `project.PT_PROJID_PRIMARY`

**Comment by daved** [comment]
Testing:
1. The gathering logic was tested on our current project to ensure only relevant stories were collected.
2. The move logic was tested using a throwaway project to avoid harming our current project.

If there is some way to back-up the project before running this, I'd feel better. Otherwise, the limit can be decreased for a first run.

File: scripts/story-cleanup/story-cleanup.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+from requests.exceptions import HTTPError
+
+PT_PROJID_PRI = os.getenv('PT_PROJID_PRIMARY')
+PT_PROJID_STR = os.getenv('PT_PROJID_STORAGE')
+PT_URLPFX_FMT = 'https://www.pivotaltracker.com/services/v5/projects/{}'
+PT_URLPFX_PRI = PT_URLPFX_FMT.format(PT_PROJID_PRI)
+PT_HDRS_BASE = {'X-TrackerToken':os.getenv('PT_API_TOKEN')}
+PT_PRMS_BASE = {'date_format':'millis', 'limit':50}
+PT_DAYS_AGED = 190 if 'PT_DAYS_AGED' not in os.environ else int(os.getenv('PT_DAYS_AGED'))
```
Comment (Naatan) [inline scripts/story-cleanup/story-cleanup.py:17]: Can you please rename these to make them a little more humanly readable.

eg. `PT_PROJID_STR` > `PT_PROJID_STORAGE`

File: scripts/story-cleanup/story-cleanup.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    response = requests.put(PT_URLPFX_PRI+'/stories/'+storyID, headers=PT_HDRS_BASE, json=payload)
+    response.raise_for_status()
+
+def main():
+    try:
+        agedStories = getAgedStories()
+        for story in agedStories:
+            moveStory(story)
+    except HTTPError, e:
+        print(e)
```
Comment (Naatan) [inline scripts/story-cleanup/story-cleanup.py:51]: Then why even catch it? :p

**Review by Naatan — CHANGES_REQUESTED** [review]

File: scripts/story-cleanup/story-cleanup.py (hunk 2 above)
Comment (daved) [reply scripts/story-cleanup/story-cleanup.py:51]: I fixed this a little, but the main reason is that the printed output is more easily readable than the trace.

**Review by Naatan — APPROVED** [review]

**Pull request merged**